docs/docs/migration/v0.22.2.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 0
+title: v0.22.2
+description: For chains that were scaffolded with Ignite CLI versions lower than v0.22.2, changes are required to use Ignite CLI v0.22.2. 
+---
+
+# Upgrading a blockchain to use Ignite CLI v0.22.2
+
+Ignite CLI v0.22.2 changed the GitHub username from "ignite-hq" to "ignite", which means the imports must be fixed to reflect this change.
+
+1. In your `go.mod` file find the require line for Ignite CLI that starts with `github.com/ignite-hq/cli` and is followed by a version.
+   It looks something like `github.com/ignite-hq/cli v0.22.0`, and replace it by `github.com/ignite/cli v0.22.2`.
+
+2. Open the `app/app.go` file and replace the import line `github.com/ignite-hq/cli/ignite/pkg/openapiconsole` by `github.com/ignite/cli/ignite/pkg/openapiconsole`.
+
+3. Open the following files and replace the import line `github.com/ignite-hq/cli/ignite/pkg/cosmoscmd` by `github.com/ignite/cli/ignite/pkg/cosmoscmd`:
+
+ - `app/app.go`
+ - `app/simulation_test.go`
+ - `testutil/network/network.go`
+ - `main.go` file located in the sub-folder inside the app's `cmd/` directory
+
+5. If you imported a package from Ignite CLI after scaffolding make sure to change those import URL starting with the prefix `github.com/ignite-hq/cli` to start
+   with the prefix `github.com/ignite/cli` instead.
+
+6. Finally run `go mod tidy` where your app's `go.sum` file is located.
```

Comment:
Why specifying where `go mod tidy` should be run ? It works also if `go mod` is run in a sub-directory of the module root directory.

I suggest to replace that with : 
```suggestion
6. Finally run `go mod tidy`,  then ensure there's no mention if `ignite-hq/cli` in your go.mod.
```